util/dvsim/Scheduler.py

```diff
@@ -164,10 +160,8 @@ def _cancel(self, item):
 
     def _check_if_done(self, timer, hms, print_status):
         '''Check whether we are finished.
-
         If print_status or we've reached a time interval then print current
         status for those jobs that weren't known to be finished already.
-
         '''
```

Comment:
This is wrong, I think. See [PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) for the correct formatting (which is what existed before the patch).

Reply:
I think that there are still some deleted lines for `_poll` and `_dispatch`. Could you drop those hunks too?